src/screens/ENSConfirmRegisterSheet.tsx

```diff
@@ -215,7 +218,18 @@ export default function ENSConfirmRegisterSheet() {
           }
         />
       ),
-      [REGISTRATION_STEPS.EDIT]: null,
+      [REGISTRATION_STEPS.EDIT]: (
+        <EditContent
+          accentColor={accentColor}
+          sendReverseRecord={sendReverseRecord}
+          setSendReverseRecord={
+            registrationCostsData?.isSufficientGasForStep
+              ? setSendReverseRecord
```

Comment:
Just double checked this. We set the switch on by default if there is no primary name set, and it is off if there is one set.

In the above screenshot, opopopop.eth has it default set to off because there is already a primary name set (estebanmino.eth)

Reply:
hm this is similar to the comment below, if i'm editing an ens the switch should be ON by default, independent on the primary name

Reply:
added a comment here https://linear.app/rainbow/issue/TEAM2-9#comment-78221d81 WDYT?

Reply:
Are we sure we want this? A bit weary that users may just update non-primary ENS names, and not even know about the switch

Reply:
Anyway, it is now fixed in the latest commit